notification-hubs-test-app-refresh/build.gradle

```diff
@@ -1,5 +1,11 @@
 apply plugin: 'com.android.application'
 
+def secretsProperties = new Properties()
+def secretsPropertiesFile = rootProject.file("secrets.properties")
+if (secretsPropertiesFile.exists()) {
+    secretsProperties.load(new FileInputStream(secretsPropertiesFile))
+}
+

```

Comment:
I think this is out of the scope for this PR. Nothing about adding SharedPreferences requires a secret, does it?

Reply:
It's required for gradlew build that we run on CI(and locally)